setup.ps1

```
@@ -449,6 +449,24 @@ if (-not $thunderPath) {
     $thunderPath = Join-Path $scriptDir 'thunder'
 }
 
+# Start Consent Server (if available)
+$consentProc = $null
+$consentDir = Join-Path $scriptDir 'consent'
+if (Test-Path $consentDir) {
+    Write-Host "[INFO] Starting Consent Server..." -ForegroundColor Cyan
```

PSScriptAnalyzer (1.24.0)

[warning] File 'setup.ps1' uses Write-Host. Avoid using Write-Host because it might not work in all hosts, does not work when there is no host, and (prior to PS 5.0) cannot be suppressed, captured, or redirected. Instead, use Write-Output, Write-Verbose, or Write-Information.

(PSAvoidUsingWriteHost)